Assets/Scripts/Actions/Food.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System.Linq;
+using UnityEngine;
+
+namespace Actions {
+    public class Food : MonoBehaviour, IAction {
+        public void execute() {
+            var item = gameObject.GetComponent<ItemAssociation>();
+            gameObject.transform.parent.gameObject.GetComponent<PlayerMovement>().Heal(item.item.healthFill);
```

Comment:
Instead of giving every item a healthFill property, I think it is easier to add the property to this script as a public variable, then you only have to see it when you make something a food